pkg/cli/build.go

```diff
@@ -1107,11 +1107,46 @@ func (t *task) buildBundle(ctx context.Context) error {
 		return nil
 	}
 
+	log.Infof("Processing results: %s", t.pkg)
+
+	tmpdir, err := os.MkdirTemp("", "")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(tmpdir)
+
+	var filemu sync.Mutex
+	filesByArch := map[string][]string{}
+
+	var pkgGroup errgroup.Group
```

Comment:
WithContext?

Reply:
I think not -- even if one arch fails, we still want to upload the other arch's packages.